src/scripts/meet.js

```diff
@@ -1,8 +1,8 @@
 const autoJoin = () => {
-  var initiateAutoJoin = setInterval(() => {
+  var initiateAutoJoin = setTimeout(() => {
```

Comment:
Any reason to change `setInterval` to `setTimeout` ?

Reply:
Not really. Just didn't find a reason as to why was it `setInterval` in the first place.

Reply:
Reset the changes in this file.

Reply:
Oh okay I get it now. `setInterval` was there incase it doesn't join in the first attempt. My bad.